code/game/objects/explosion.dm

```diff
@@ -37,7 +37,10 @@
 
 		var/start = world.timeofday
 		epicenter = get_turf(epicenter)
-		if(!epicenter) return
+		if(!epicenter)
+			return
+
+		score["explosions"]++ //For the scoreboard

```

Comment:
You should probably scale this, an assistant with an IED is AS bad as a fucking 7dev